johnwayodi/store-manager
# #161078765 Add Shopping Cart View

#### What does this PR do?
Adds Shopping Cart Page
#### Description of Task to be completed?
Enable Store Attendant to remove products from the Cart and also allow the Store attendant to create a Sale Record 
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3. `git checkout ft-product-details-161078989`
4. open **_product-detail.html_** on browser
#### Any background context you want to provide?
The Shopping cart is where the Store attendant can remove products before finalizing the creation of a sale record
#### What are the relevant pivotal tracker stories?
#161078765
#### Screenshots (if appropriate)
![shopping-cart](https://user-images.githubusercontent.com/10683903/46749989-a2fb5400-ccbf-11e8-8dbe-3605c50376e7.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
